README.md

```diff
@@ -3147,6 +3147,299 @@ $ curl -s "https://mainnet.aeternity.io/mdw/v2/transfers?account=ak_7myFYvagcqh8
 
 ---
 
+## NFTs (AEX-141 contracts and tokens)
+
+AEX-141 NFT contracts might organize the access and storage of NFTs metadata in flexible ways. This behaviour is declared during contract creation with the metadata_type field. This and other meta_info fields like name and symbol can be accessed by `/aex141` endpoint that displays information about NFT contracts. 
+
+With other endpoints it's possible to know more about AEX-141 NFTs including:
+
+- who is the owner of an NFT
+- who are the nft owners on a collection
+- which NFTs a wallet owns
+- NFT transfers filtered by contract and/or wallet
+- and more to come ... 
+
+### `/v2/aex141`
+
+Returns creation and stats information in default paginated way for all NFT collection. The same general paginated parameters might be used:
+
+- `direction` with `forward` value ascendinly sorts the response contracts by creation transaction index while `backward` works for desc sorting
+- `limit` defines the page size
```

Comment:
Perhaps we can just say that these are paginated endpoint to avoid repeating that there's `limit` and `direction` which applies for all paginated endpoints?

Reply:
Done